### PR TITLE
feat: build ussuri release on core20

### PR DESCRIPTION
**Pull request by mert-kirpici**
Building on core22 causes errors starting python-tempestconf

To be merged after #39

### CODE REVIEW

**Comment by mert-kirpici** [comment]
Test [build](https://launchpad.net/~mertkirpici/snap-tempest/+snap/snap-tempest-ussuri-test) on various archs.

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -87,9 +88,11 @@ parts:
     - cargo
     - pkg-config
     - git
+    override-stage: |
+      snapcraftctl stage
+      snapcraftctl set-version "$(tempest --version | awk '{print $2}')"
```
Comment (agileshaw) [inline snap/snapcraft.yaml:93]: I'm curious about why we are moving the version setting to staging step.

Comment (mert-kirpici) [reply snap/snapcraft.yaml:93]: doing this in prime step resulted in the following error:
```
error: 'PARTS_CALL_FIFO' environment variable must be defined.                                                        
Note that this utility is designed for use only in part scriptlets.                                                   
Failed to run 'override-prime': Exit code was 1.
```
This does not happen in staging step.

Comment (agileshaw) [reply snap/snapcraft.yaml:93]: That's interesting. In that case, can we also move the bash completion setup to the staging step (or in building stage like [this](https://github.com/canonical/charmed-openstack-upgrader/blob/main/snap/snapcraft.yaml))? I just think there's no need overriding two steps when we can do it in a single place.

Comment (mert-kirpici) [reply snap/snapcraft.yaml:93]: Just copying all to stage step also produces errors. This is a very minor thing that should cause no concern, really.

However, if you would like to see it done that way, please feel free to close this PR.

**Review by aieri — APPROVED** [review]

File: snap/snapcraft.yaml (hunk 1 above)
Comment (agileshaw) [reply snap/snapcraft.yaml:93]: Though I prefer having them in a single step, it is not blocker.

**Review by agileshaw — APPROVED** [review]

**Pull request merged**